jina/drivers/multimodal.py

```diff
@@ -83,11 +68,11 @@ def _apply_all(
 
         valid_docs = []
         for doc in docs:
-            doc_content = _extract_modalities_from_document(doc)
```

Comment:
Maybe we can adapt the interface to be `docs: MultiModalDocumentSet`, would that make sense?

Reply:
my understanding is that `DocumentSet` should handle both `Document` and `MultimodalDocument`.